.travis.yml

```diff
@@ -1,55 +1,114 @@
+sudo: false
+dist: xenial
 language: python
+notifications:
+  email: false
 python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7
 
 stages:
   - test
   - moban
 
 .disable_global: &disable_global
+  addons: false
+  cache: false
+  env: {}
+  python: false
   before_install: false
   install: false
+  before_script: false
   script: false
+  after_success: false
+  after_failure: false
+  before_deploy: false
+  deploy: false
 
-.check_moban: &check_moban
+.moban: &moban
   <<: *disable_global
   python: 3.6
   stage: moban
-  install: pip install moban
-  script: .ci/check_moban.sh
+  install: pip install moban>=0.0.4
+  script:
+    - moban
+    - git diff --exit-code
```

Comment:
What is this command for? It's causing the moban stage in travis to fail to fail. https://travis-ci.org/coala/coala-quickstart/jobs/567742606

Seems like it's there to make sure there are no local unstaged changes after running moban.

Reply:
@satwikkansal It is to ensure that the all the target files are in sync with our templates. So, any change in the templates would cause the build to fail unless the target files are updated too as per the changes made to the templates. You can run `moban` to update all the files.

Reply:
This line was previously in `check_moban.sh` https://github.com/coala/coala-quickstart/pull/338/files#diff-417fc13c51865be1380a0a43d8cdf373L15

The job is allowed to fail on PRs, but not on master, so the cron jobs show when the repo is out of sync, but it doesnt stop PRs/development.